crates/viewer/re_view_spatial/src/max_image_dimension_subscriber.rs

```diff
@@ -170,39 +212,54 @@ impl PerStoreChunkSubscriber for MaxImageDimensionsStoreSubscriber {
     }
 }
 
-fn size_from_blob(
+fn try_size_from_blob(
     blob: &[u8],
-    media_type: &MediaType,
-    entity_path: &EntityPath,
+    media_type: Option<components::MediaType>,
+    archetype_name: ArchetypeName,
+    debug_name: &str,
 ) -> Option<[u32; 2]> {
     re_tracing::profile_function!();
 
-    if media_type.is_image() {
+    if archetype_name == archetypes::EncodedImage::name() {
```

Comment:
This will work until we add `EncodedDepthImage` (#9046). Might be good to test for the `media_type` as well, as this seems like a place to break without going noticed.